### DISCUSSION TITLE
Help With these errors

### DISCUSSION THREAD

**Discussion by lostedward** [jfarmer08/homebridge-wyze-smart-home#187] [Q&A]
10/25/2023, 8:36:07 PMWyze Smart Home BridgeWyzeERRORError getting devices: AxiosError: Request failed with status code 403
10/25/2023, 8:36:10 PMWyze Smart Home BridgeConfiguration change
10/25/2023, 8:36:10 PMwyzesmarthomebridge stopping
10/25/2023, 8:36:15 PMWyze Smart Home Bridge starting
10/25/2023, 8:36:24 PMWyze Smart Home BridgeLoaded plugin 'homebridge-wyze-smart-home'
10/25/2023, 8:36:25 PMWyze Smart Home BridgeLoading 1 platforms...
10/25/2023, 8:36:25 PMWyze Smart Home BridgeBridge is running on port 51856.
10/25/2023, 8:36:26 PMWyze Smart Home BridgeWyzeERRORError getting devices: Error: ENOENT: no such file or directory, open '/var/lib/hoobs/wyzesmarthomebridge/persist/wyze-0bab04c5-aa69-5483-9f5a-55a9aa68561b.json'

**Comment by jfarmer08** [comment]
You need to enable logging.
